src/ca/mcgill/cs/jetuml/gui/ToolBar.java

```diff
@@ -264,13 +263,12 @@ public void actionPerformed(ActionEvent pEvent)
 	
 	private void copyToClipboard()
 	{
-		// Obtain the editor frame by going through the component graph
-		Container parent = getParent();
```

Comment:
Obtaining through the graph should become possible again if `EditorFrame` is made a `JPanel`.